organisaatio-ui/src/components/Sivut/LomakeSivu/LomakeSivu.tsx

```diff
@@ -143,13 +144,18 @@ const LomakeSivu = ({ match: { params }, history }: LomakeSivuProps) => {
 
     useEffect(() => {
         (async function () {
-            const organisaatio = await readOrganisaatio(params.oid);
-            if (organisaatio) {
-                await resetOrganisaatio({ ...organisaatio });
+            try {
+                const organisaatio = await readOrganisaatio(params.oid);
```

Comment:
does this readOrganisaatio twice when name is changed?

Reply:
no it triggers only once, i tested when developing.

Reply:
if only this could be repeated/verified automatically...

Reply:
if you want i will use more time to try to figure out how to test this? now i have no idea.

Reply:
..that was just a remark